cmd/harmony/flags.go

```diff
@@ -392,20 +392,20 @@ func applyNetworkFlags(cmd *cobra.Command, cfg *harmonyConfig) {
 	}
 
 	if cli.IsFlagChanged(cmd, dnsZoneFlag) {
-		cfg.Network.DNSZone = cli.GetStringFlagValue(cmd, dnsZoneFlag)
+		cfg.DNSSync.Zone = cli.GetStringFlagValue(cmd, dnsZoneFlag)
```

Comment:
Since the field has already been changed, do you think it would be better to aggragate the dns related flags to the same catagory?